__tests__/src/components/ViewerNavigation.test.js

```diff
@@ -16,125 +15,138 @@ function createWrapper(props) {
-      expect(next.style).toEqual({ transform: 'rotate(180deg)' });
+      expect(screen.getByLabelText('previousCanvas', { selector: 'button' }).querySelector('svg')).not.toHaveStyle('transform: rotate(180deg);'); // eslint-disable-line testing-library/no-node-access
+      expect(screen.getByLabelText('nextCanvas', { selector: 'button' }).querySelector('svg')).toHaveStyle('transform: rotate(180deg);'); // eslint-disable-line testing-library/no-node-access
     });
 
     it('sets the dir="rtl"', () => {
-      expect(wrapper.find('div').props().dir).toBe('rtl');
+      createWrapper({
+        hasNextCanvas: true,
+        hasPreviousCanvas: true,
+        setNextCanvas,
+        setPreviousCanvas,
+        viewingDirection: 'right-to-left',
+      });
+      expect(screen.getByRole('navigation', { selector: 'div' })).toHaveAttribute('dir', 'rtl');
     });
   });
 
   describe('when viewingDirection is top-to-bottom', () => {
-    beforeEach(() => {
-      wrapper = createWrapper({
+    it('changes the arrow styles', () => {
+      createWrapper({
         hasNextCanvas: true,
         hasPreviousCanvas: true,
         setNextCanvas,
         setPreviousCanvas,
         viewingDirection: 'top-to-bottom',
       });
-    });
-
-    it('changes the arrow styles', () => {
-      const previous = wrapper.find(MiradorMenuButton).first().children(NavigationIcon).props();
-      const next = wrapper.find(MiradorMenuButton).last().children(NavigationIcon).props();
-      expect(previous.style).toEqual({ transform: 'rotate(270deg)' });
-      expect(next.style).toEqual({ transform: 'rotate(90deg)' });
+      expect(screen.getByLabelText('previousCanvas', { selector: 'button' }).querySelector('svg')).toHaveStyle('transform: rotate(270deg);'); // eslint-disable-line testing-library/no-node-access
+      expect(screen.getByLabelText('nextCanvas', { selector: 'button' }).querySelector('svg')).toHaveStyle('transform: rotate(90deg);'); // eslint-disable-line testing-library/no-node-access
     });
   });
-
   describe('when viewingDirection is bottom-to-top', () => {
-    beforeEach(() => {
-      wrapper = createWrapper({
+    it('changes the arrow styles', () => {
+      createWrapper({
         hasNextCanvas: true,
         hasPreviousCanvas: true,
         setNextCanvas,
         setPreviousCanvas,
         viewingDirection: 'bottom-to-top',
       });
-    });
-
-    it('changes the arrow styles', () => {
-      const previous = wrapper.find(MiradorMenuButton).first().children(NavigationIcon).props();
-      const next = wrapper.find(MiradorMenuButton).last().children(NavigationIcon).props();
-      expect(previous.style).toEqual({ transform: 'rotate(90deg)' });
-      expect(next.style).toEqual({ transform: 'rotate(270deg)' });
+      expect(screen.getByLabelText('previousCanvas', { selector: 'button' }).querySelector('svg')).toHaveStyle('transform: rotate(90deg);'); // eslint-disable-line testing-library/no-node-access
```

Comment:
I think checking the role is probably preferable to checking by text

```suggestion
      expect(screen.getByRole('button', { name: 'previousCanvas' }).querySelector('svg')).toHaveStyle('transform: rotate(90deg);'); // eslint-disable-line testing-library/no-node-access
```

Reply:
@cbeer: I don't see a button role on anything in the debug? Changing the line to what you suggested makes the test fail.

Reply:
Unless the button has an explicit role, it should have the implicit role `button`:

Reply:
@cbeer I was able to update them all to use the button role.